### DISCUSSION TITLE
Additional Parameters on Invite User By Email

### DISCUSSION THREAD

**Discussion by DKBoston15** [supabase/supabase#2171] [Questions] [answered]
I am developing an app where multiple users share a workspace ID. I'd like to include that workspace ID in the url sent in the invite user email as a parameter in order to assign the user to the right workspace. Anyone know if this is possible?

**Comment by inian** [comment, marked as answer]
I think you can modify the email template and add in the workspace id - https://app.supabase.io/project/your-project-ref/auth/templates and change the template to have `"{{ .ConfirmationURL }}&workspace_id=123"`.  That should probably work. 

My initial thought is to handle this outside of the invite process though. You can verfiy the user through the usual flow and then add verified users to the right workspace after that. Relying on the workspace_id directly from the URL is probably not a good idea security wise too (unless they are unguessable)
